index.html

```diff
@@ -1,12 +1,118 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Layout3 using grid</title>
-    <link rel="stylesheet" href="index.css">
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Layout 3 using grid</title>
+    <link rel="stylesheet" href="index.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css"
+      integrity="sha512-DTOQO9RWCH3ppGqcWaEA1BIZOC6xxalwEsw9c2QQeAIftl+Vegovlnee1c9QX4TctnWMn13TZye+giMm8e2LwA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <div class="main">
+      <div class="wrapper">
+        <div class="navbar">
+          <div class="nav">
+            <div class="item1">Layout 3</div>
+            <div class="item2">About</div>
+            <div class="item3">Services</div>
+            <div class="item3">Contact</div>
+          </div>
+        </div>
+        <div class="sub_wrapper">
+            <div class="left">
+              <div class="blog_post_title">Blog Post Title</div>
+              <div class="by_test">by <inline class="test">Test</inline></div>
+
+            <div class="padding_date">
+                <div class="post_date">
+                    <i class="fa-regular fa-clock"></i>Posted on August 24, 2013 at
+                        9:00 PM
+                      </div>
+            </div>
+             
+
+
+              <div class="img_div">
+                <div class="img">900 x 300</div>
+              </div>
+              <div class="paragraphs">
+                <div class="weight_para">
+                  Lorem ipsum dolor sit amet consectetur adipisicing elit. Modi
+                  culpa doloremque impedit iste, praesentium dolore, consectetur
+                  sapiente dignissimos earum deserunt!
+                </div>
+                <p class="para1 para">
```

Comment:
use genuine name of class name,
if para1 class is not used why unnecessary provided